src/Main/TeamContainer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import TeamMember from './TeamMember'
+import Team from '../assets/team'
+
+const TeamContainer = (props) => {
+  return (
+    <div className="team-container">
```

Comment:
The font-size of this component seems a bit too small compared to the text above it. Can we make a bit bigger?